lib/Login.dart

```diff
@@ -8,65 +9,110 @@ class Login extends StatefulWidget {
 }
 
 class _LoginState extends State<Login> {
-
   @override
   void initState() {
     super.initState();
   }
 
   @override
-  Widget build(BuildContext context) {
+  Widget build(context) {
     AuthProvider authProvider = Provider.of(context);
     try {
       authProvider.signInSilently();
     } catch (e) {
-      print('User has not logged in previously, therefore, we should not proceed');
+      print(
+          'User has not logged in previously, therefore, we should not proceed');
     }
     return Scaffold(
-        body: Container(
-            color: Colors.white,
-            child: Center(
-                child: Column(
-              mainAxisSize: MainAxisSize.max,
-              mainAxisAlignment: MainAxisAlignment.center,
-              children: <Widget>[
-                FlutterLogo(size: 150),
-                SizedBox(height: 50),
-                SignInButton()
-              ],
-            ))));
+      backgroundColor: Colors.black,
+      body: Container(
+        margin: EdgeInsets.only(bottom: 20),
+        height: MediaQuery.of(context).size.height,
+        child: Column(
+//        mainAxisSize: MainAxisSize.max,
+          children: <Widget>[
+            SizedBox(
+              height: MediaQuery.of(context).size.height / 6,
+            ),
+            Container(
+              margin: EdgeInsets.only(left: 20),
+              child: Row(
+                mainAxisAlignment: MainAxisAlignment.start,
+                children: [
+                  Text('Welcome to Carriage',
+                      style: TextStyle(fontSize: 33, color: Colors.white)),
+                ],
+              ),
+            ),
+            SizedBox(height: 20),
+            Container(
+              margin: EdgeInsets.only(left: 20),
+              child: Row(
+                mainAxisAlignment: MainAxisAlignment.start,
+                children: [
+                  Text('Sign in using your Cornell email',
+                      style: TextStyle(fontSize: 15, color: Colors.white54)),
+                ],
+              ),
+            ),
+            SizedBox(height: 60.0),
+            Image.asset(
+              'assets/images/Logo-No text.png',
```

Comment:
I don't think the most recent designs have the logo on this page. Thus, it would be best to remove them